gnosis/verification.k

```diff
@@ -261,4 +275,6 @@ module VERIFICATION
 
     rule #ceil32(X) => X requires X modInt 32 ==Int 0
 
+    rule X <=Int #ceil32(X) => true
```

Comment:
@yzhang90 this is not consistent with the `#ceil32` definition, when X is negative, since the definition uses `/Int` instead of `modInt`.

Reply:
fixed